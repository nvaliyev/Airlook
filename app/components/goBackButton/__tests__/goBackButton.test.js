import 'react-native';
import React from 'react';
import ShallowRenderer from 'react-test-renderer/shallow';
import GoBack from '..';
describe('GoBack Button', () => {
    test(' Goback Button  snapshot', () => {
        const renderer = new ShallowRenderer();
        const snapshot = renderer.render(<GoBack/>);
        expect(snapshot).toMatchSnapshot();
    });
});


