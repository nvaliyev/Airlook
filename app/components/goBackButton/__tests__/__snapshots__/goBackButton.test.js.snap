// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GoBack Button  Goback Button  snapshot 1`] = `
<Themed.Icon
  color="#F7F9FC"
  name="navigate-before"
  onPress={[Function]}
  size={40}
/>
`;
